### ISSUE TITLE
ms-rest 1.15.3 npm includes wrong-case webresource.js

### ISSUE THREAD

**Issue by youngbupark** [Azure/azure-sdk-for-node#2042]
We found that ms-rest npm has been updated from 1.15.2 to 1.15.3 today. 
msRest.js requires lib/webResource.js, but it is failing to find it in linux environment due to the wrong filename case.
1.15.3 npm package includes webresource.js, not webResource.js

Please fix this issue as soon as possible.

Thanks, young.

**Comment by markmeeus** [comment]
This is a serious issue for us as well. +1 on giving this some priority

**Comment by GlennGeenen** [comment]
All our builds are broken because of this!!!

**Comment by me97esn** [comment]
+1

**Comment by me97esn** [comment]
For those of you that want a workaround until Ms fixes the bug, this is how I solved it:
Start using Yarn in the project
`npm i yarn -g`
Install the exact dependency for the latest working ms-rest version
`yarn add ms-rest@1.5.1 --ignore-engines`

Open the file yarn.lock and replace the following lines:

```yaml
ms-rest@^1.14.0, ms-rest@^1.14.2, ms-rest@^1.14.3, ms-rest@^1.15.0, ms-rest@^1.15.2:
  version "1.15.3"
  resolved "https://registry.yarnpkg.com/ms-rest/-/ms-rest-1.15.3.tgz#e984f654b2d44ee12634c82fa8f98690eeeb2c85"
  dependencies:
    duplexer "~0.1.1"
    moment "^2.14.1"
    request "~2.74.0"
    through "~2.3.4"
    tunnel "~0.0.2"
    uuid "^3.0.1"
```
with these lines:
```yaml
ms-rest@^1.14.0, ms-rest@^1.14.2, ms-rest@^1.14.3, ms-rest@^1.15.0, ms-rest@^1.15.2:
  version "1.15.1"
  resolved "https://registry.yarnpkg.com/ms-rest/-/ms-rest-1.15.1.tgz#1f0e30da30369e2960e495381a70fe9b99a86889"
  dependencies:
    duplexer "~0.1.1"
    moment "^2.14.1"
    request "~2.74.0"
    through "~2.3.4"
    tunnel "~0.0.2"
    underscore "^1.4.0"
```

`run yarn install --ignore-engines`

Hopefully it works after this. When this bug is solved its safe to just delete the yarn.lack file and continue as usual.

**Comment by amarzavery** [comment]
I am extremely sorry for this. We have published 1.15.4 version of ms-rest and "lib/webResource.js" has the correct casing. Let me know if you still find any issues.

**Comment by balajikris** [comment]
this should now be resolved. I'm closing this issue. Please re-open if this still happens.
